digitalpy/core/zmanager/routing_proxy.py

```diff
@@ -1,7 +1,7 @@
 import zmq
 import multiprocessing
 from digitalpy.core.object_factory import ObjectFactory
-from digitalpy.config.configuration import Configuration
+from digitalpy.core.digipy_configuration.configuration import Configuration
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I001 isort found an import in the wrong position